test/integration/database_schema_test.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+SingleCov.not_covered!
+
+describe "database schema" do
+  attr_reader :table_definitions
+
+  before(:all) do
```

Comment:
I don't think that works ... not sure what the first argument does though 🤷‍♂ 

`let_all ` would work ... but if it's not too slow it does not matter either